index.html

```diff
@@ -0,0 +1,258 @@
+    <a class="connect">LET'S CONNECT</a>
+    <ul class="social-links">
+      <li><a href="https://twitter.com/obibaadoma">
+        <img src="images\twitter.png" alt="Twitter icon" class="icon"/></a></li>
+      <li><a href="https://www.linkedin.com/in/derek-akrasi-konadu-187453151/">
+        <img src="images\linkedin.png" alt="LinkedIn icon" class="icon"/></a></li>
+      <li><a><img src="images\medium.png" alt="Medium icon" class="icon"/></a></li>
+      <li><a href="https://github.com/obibaadoma">
+        <img src="images\git.png" alt="GitHub icon" class="icon"/></a></li>
+      <li><a><img src="images\angelist.png" alt="Angelist icon" class="icon"/></a></li>
+    </ul>
+    <button type="submit" class="main-buttons">Get my resume</button>
+  </div>
+  <ul class="about-myself-ul">
+    <li class="dropdowns" id="open">
+      <a>Languages</a>
+      <button type="submit"><img src="images\Icon-Chevron-Down-Larger.png" 
+        alt="Dropdown button"></button>
+    </li>
+    <li>
+      <ul class="languages-ul">
+        <li class="languages" id="js">JavaScript</li>
+        <li class="languages" id="html">HTML</li>
+        <li class="languages" id="css">CSS</li>
+      </ul>
+    </li>
+    <li class="dropdowns">
+      <a>Frameworks</a>
+      <button type="submit" ><img src="images\Disabled.png" alt="Dropdown button"></button>
+    </li>
+    <li class="dropdowns">
+      <a>Skills</a>
+      <button type="submit"><img src="images\Disabled.png" alt="Dropdown button"></button>
+    </li>
+  </ul>
+</section>
+
+<!-- Contact Section -->
+<section class="contact-section" id="Contact">
+  <div class="contact-container">
+    <div class="form-container">
+      <h2>Contact me</h2>
+      <p>If you have an application you are interested in developing, 
+        a feature that you need built or a project that needs coding. 
+        I’d love to help with it!</p>
+      <form action="https://formspree.io/f/mbjevloq" method="POST">
+        <input 
+          type="text" 
+          class="input-container" 
+          placeholder="Enter your name" 
+          maxlength="30" 
+          name="fullname" 
+          id="fullname" 
+          required
+        >
+        <label class="labels" for="fullname">Full name</label>
+        <input 
+          type="email" 
+          class="input-container" 
+          placeholder="doejones@obiba.com" 
+          name="email" 
+          id="email" 
+          required
+        >
+        <label class="labels" for="email">Email address</label>
+        <textarea 
+          type="text" 
+          class="input-container-2" 
+          placeholder="Write your message here..." 
+          maxlength="500" 
+          name="comment" 
+          id="comment" 
+          required
+        ></textarea>
+        <label class="labels" for="comment">Comment</label>
+        <button type="submit" class="button-form">Get in touch</button>
+      </form>
+    </div>
+  </div>
+</section>
```

Comment:
- [ ] As per requirements to stick to the `Figma` design as much as possible, kindly refer back to the template to use the same fonts as per the specs.

 | Your design | Figma Design |
| ------------- | ------------- |
| ![Screenshot 2023-06-10 214723](https://github.com/obibaadoma/potfolio-desktop/assets/102612396/8b5382b3-c4fd-464b-9ad8-ff0479ab966b) | ![image](https://github.com/obibaadoma/potfolio-desktop/assets/102612396/afb6dfed-8877-4a84-890c-4ec33adfc40b) |

Reply:
done